scripts/pre_commit_linter.py

```diff
@@ -357,11 +357,14 @@
     os.path.join(_PARENT_DIR, 'oppia_tools', 'PyGithub-1.43.5'),
     os.path.join('third_party', 'backports.functools_lru_cache-1.5'),
     os.path.join('third_party', 'gae-pipeline-1.9.17.0'),
-    os.path.join('third_party', 'bleach-1.2.2'),
+    os.path.join('third_party', 'bleach-3.1.0'),
     os.path.join('third_party', 'beautifulsoup4-4.7.1'),
     os.path.join('third_party', 'gae-mapreduce-1.9.17.0'),
     os.path.join('third_party', 'mutagen-1.38'),
-    os.path.join('third_party', 'soupsieve-1.8'),
+    os.path.join('third_party', 'soupsieve-1.9.1'),
+    os.path.join('third_party', 'six-1.12.0'),
+    os.path.join('third_party', 'webencodings-0.5.1'),
+    os.path.join('third_party', 'callbacks-0.3.0'),
```

Comment:
Ditto. (only the third party libs)

Reply:
Done, Thanks.

Reply:
Done, Thanks.

Reply:
Also @apb7, I'll again re-arrange all the libs at the end of the project

Reply:
It would be better to just keep them up to date each time, I think.

Reply:
Hi @seanlip, Yes I'll take care